prompting/utils/config.py

```diff
@@ -71,6 +71,20 @@ def add_args(cls, parser):
         help="Device to run on.",
         default="cuda" if torch.cuda.is_available() else "cpu",
     )
+    
+    parser.add_argument(
+        "--neuron.gpus",
+        type=int,
+        help="The number of visible GPUs to be considered in the llm initialization. This parameter currently reflects on the property `tensor_parallel_size` of vllm",
+        default=1,
+    )
+    
+    parser.add_argument(
+        "--neuron.llm_max_allowed_memory_in_gb",
```

Comment:
Can we use this arg from `min_compute.yaml`?

Reply:
I don't  think people use the `.yaml` to run the validator as a config but I totally agree that we should update `min_compute.yaml` and `README.md`